Unix/share/networkschema/MSFT_ACL.mof

```diff
@@ -19,12 +19,12 @@ class MSFT_ACL : CIM_NetworkPolicyRule
 
 [Description ( 
           "Set of policy on an ACL"
-		  "The default Implicited states that all traffic is denied unless there is a"
+		  "The default Implicit states that all traffic is denied unless there is a"
 		  "rule allowing traffic"
-		  "Explicit policy states that an explict rule must be stated for each permit and or allow action"), 
+		  "Explicit policy states that an explicit rule must be stated for each permit and or allow action"), 
        ValueMap { "2", "3"}, 
        Values { "Implicit", "Explicit"}
 
 	]
    uint16 ActionPolicy = 2;
-};
```

Comment:
this change is because i manually touch the file. if it's a problem, I can fix..